src/redux/actionTypes/experimentSettings.js

```diff
@@ -1,5 +1,10 @@
 const EXPERIMENT_SETTINGS = 'experimentSettings';
```

Comment:
you can delete this `src/pages/notneeded.jsx ` file too

Reply:
Done.